dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.42/V2_42_29__add_2FA_type_userinfo.sql

```diff
@@ -0,0 +1,8 @@
+-- DHIS2-13334: Add twofactortype column to userinfo table
+ALTER TABLE userinfo
+    ADD COLUMN IF NOT EXISTS twofactortype character varying(50);
+
+-- Set all existing users to have the default twofactortype TOTP, if the secret column is not empty.
+UPDATE userinfo
+SET twofactortype = 'TOTP'
```

Comment:
does this value need to match the updated enum value?

Reply:
Thanks, updated.